EWPP-4642: Update to Drupal 11.

kp77 opened the pull request:
## EWPP-4642

### Description

Add Drupal 11 support.

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

kp77 committed:
EWPP-4642: Remove openeuropa/code-review-drupal repository.

The package has been published to packagist.

kp77 committed:
EWPP-4642: Remove openeuropa/DrupalDriver repository.

The original jhedstrom/DrupalDriver package version 2.3 supports D11.

kp77 committed:
EWPP-4642: Update composer requirements for D11 support.

kp77 committed:
EWPP-4642: Use drupal/core-dev instead of requiring phpunit-bridge.

kp77 committed:
EWPP-4642: Add D11 support in info.yml files.

kp77 committed:
EWPP-4642: Update `chromeOptions` key to `goog:chromeOptions` for phpunit and behat.

See: https://www.drupal.org/node/3422624 for details.

kp77 committed:
EWPP-4642: Generate settings.testing.php in runner used for functional tests.

kp77 committed:
EWPP-4642: Fix PHPUnit errors throwing `SessionNotFoundException`.

kp77 committed:
EWPP-4642: Fix type error passing int to ConstraintViolationBuilder::atPath().

Passing a string $path argument to ConstraintViolationBuilder::atPath($path)
is required in Drupal 11 (https://www.drupal.org/node/3396238).

kp77 committed:
EWPP-4642: Rename classes ending in `Test` but not extending PHPUnit\Framework\TestCase.

Fixes PHPUnit warnings.

kp77 committed:
EWPP-4642: Pass mandatory typed config manager in PurlSettingsForm constructor.

kp77 committed:
EWPP-4642: Remove category property from field type definitions.

kp77 committed:
EWPP-4642: Fix NodeFormRedirectLinkTest, node type title passed as null.

kp77 committed:
EWPP-4642: Fix missing schema errors for oe_content_persistent_test editor config.

kp77 committed:
EWPP-4642: Fix PHPUnit deprecation Data Provider method is not static.

kp77 committed:
EWPP-4642: Adopt new OOP hook implementation for oe_content_entity_base_field_info().

kp77 committed:
EWPP-4642: Fix behat errors due to core date formats changed in D11.1.

See: https://www.drupal.org/node/3467774 for details.

kp77 committed:
EWPP-4642: Fix behat errors due to no access to media edit page.

The tests check if a link to media edit page is present on the page.
Due to core access check fix in EntityReferenceLabelFormatter the link
is not present if the user doesn't have permission to access the media
edit form.
See: https://www.drupal.org/project/drupal/issues/3386313

kp77 committed:
EWPP-4642: Add Drupal 11.1 to drone matrix.

kp77 committed:
EWPP-4642: Remove duplicate entries of GeneralContactContext from behat.yml.dist.

nagyad commented on composer.json:
```diff
@@ -7,45 +7,44 @@
         "drupal/maxlength": "^3.0.0-beta1",
-        "openeuropa/oe_media": "^1.23.1",
-        "openeuropa/rdf_skos": "^1.0.0-alpha10"
+        "drupal/media_avportal": "dev-3431855-automated-drupal-11",
```
This can be changed to a release.

kp77 replied:
Do you mean media_avportal? As I see, the [latest release](https://www.drupal.org/project/media_avportal/releases/8.x-1.10) doesn't add D11 support yet.

kp77 replied:
This is fixed, media_avportal has been removed from here.

kp77 committed:
EWPP-4642: Fix PHPUnit errors for `TimelineWidgetTest::testWidgetErrors()`.

The `$arrayPropertyPath` property was always returning null in
`TimelineFieldWidget::errorElement()`.
See: https://www.drupal.org/node/3307919

kp77 committed:
EWPP-4642: Fix PHPUnit error: no AJAX requests to wait for.
